### DISCUSSION TITLE
Thousands of lines with "Empty Chunk Data"...

### DISCUSSION THREAD

**Discussion by within7** [kubilus1/autoortho#287] [General]
When XP starts "Reading new scenery files." this is what I get... ALL THE TIME with thousands of lines. Until XP launches.

What is this and why is this happening?

![image](https://github.com/kubilus1/autoortho/assets/71191855/5b42b119-957f-4120-a347-05792848f0cb)

**Comment by dpdurst** [comment]
The answer is on that screen, server time out errors and not being able to access those servers. I've seen this before seems to happen more outside the NA area then it does in the NA area.

**Comment by drmichaelgorski** [comment]
@dpdurst do you have a solution for that problem, because I have the same issue in Germany?

**Reply by d41k4n** [reply]
IMO there is no solution to this since it is out of anyone's control. The third-party imagery providers AO relies on can throttle image requests and/or blacklist your IP anytime and without warning in order to ensure their self-defined minimum service quality thresholds are met and prevent abuse due to excessive bandwidth hogging. Most likely this is what's happening in your case. The differences seen between regions (i.e. real-world user location) are likely due to different CDN endpoints accessed through geolocation-dependant DNS.
